### PR TITLE
refactor(rxUnauthorizedInterceptor): rx-prefixes update

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-1163

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM

### CODE REVIEW

File: src/utilities/rxUnauthorizedInterceptor/docs/rxUnauthorizedInterceptor.meta.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "displayName": "rxUnauthorizedInterceptor",
+    "stability": "stable",
+    "description": "Redirects users to the login page, when user authentication fails during a system service request.",
+    "api": "service:rxUnauthorizedInterceptor",
+    "keywords": [
+        "UnauthorizedInterceptor"
```
Comment (lentusignavus) [inline src/utilities/rxUnauthorizedInterceptor/docs/rxUnauthorizedInterceptor.meta.json]: Add `rxUnauthorizedInterceptor` to keywords for searching, yes/no, maybe so? @CITguy @cathysiller

Comment (cathysiller) [inline src/utilities/rxUnauthorizedInterceptor/docs/rxUnauthorizedInterceptor.meta.json]: oh yeah i had talked about this with Jay on all the PRs @CITguy, should rx be included on keywords?

Comment (CITguy) [inline src/utilities/rxUnauthorizedInterceptor/docs/rxUnauthorizedInterceptor.meta.json]: Unneeded. Search logic will use the utility name as part of the search, so `rxUnauthorizedInterceptor` is already included.

**Comment by lentusignavus** [comment]
DEV LGTM

**Pull request merged**